### ISSUE TITLE
Phantomjs cannot read all cells in rtl webpage table

### ISSUE THREAD

**Issue by HamidMousavi** [ariya/phantomjs#14827]
1. Which version of PhantomJS are you using? Tip: run `phantomjs --version`.
phantomjs-2.1.1-windows

2. Which operating system are you using?
windows 10

3. Did you use binary PhantomJS or did you compile it from source?
binary

4. Please provide any additional information below.
 this is example html  page source:

```
<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
</head>
<body style="margin: 0 0 0 0; background: background" dir="rtl">

<table rules="all" dir="ltr" designtimedragdrop="36" id="DataGrid1" style="color:#333333;border-color:Gray;font-family:Tahoma;font-size:10pt;font-weight:bold;width:780px;border-collapse:collapse;" align="Center" border="1" cellpadding="4" cellspacing="0">
<tbody>
    <tr>
           <td>1</td>
           <td>2</td> 
           <td>3</td>
           <td>4</td>
           <td>5</td> 
           <td>6</td>
           <td>7</td>
           <td>8</td> 
           <td>9</td>
  </tr>
  </table>

</body></html>
```
![cells](https://cloud.githubusercontent.com/assets/16390069/23015904/dff44dc6-f449-11e6-8ce2-9f623ec177c8.png)

This is my c# code to scraping above page:

```
 static void Main(string[] args)
        {
            var driver = new PhantomJSDriver();
            driver.Navigate().GoToUrl("file:///C:/Users/MyName/Desktop/table.html");

            IWebElement table = driver.FindElement(By.Id("DataGrid1"));
            List<IWebElement> rows = table.FindElements(By.TagName("tr")).ToList();

            foreach (var row in rows)
            {
                List<IWebElement> cells = row.FindElements(By.TagName("td")).ToList();
                Console.WriteLine("Cells Count: " + cells.Count);

                foreach (var cell in cells)
                {
                    Console.WriteLine(cell.Text);
                }
                Console.WriteLine();
            }
        }
```

**Comment by jesg** [comment]
@HamidMousavi i can reproduce the issue.  it is a bug in ghostdriver.  you can work around the issue by increasing the screen size.

here is the screenshot.
![screenshot](https://cloud.githubusercontent.com/assets/1785589/23023233/63a67022-f409-11e6-8469-da28bfef05d9.png)

```
from __future__ import print_function
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
import time

driver = webdriver.PhantomJS()
driver.get('file:///home/jesg/cell-text.html')
# driver.maximize_window()

table = driver.find_element_by_id('DataGrid1')
cells = table.find_elements_by_tag_name('td')

for cell in cells:
    print('---')
    print(cell.text)
    print(driver.execute_script('return arguments[0].innerText;',cell))

driver.quit()
```

which returns
```
---

1
---

2
---

3
---

4
---
5
5
---
6
6
---
7
7
---
8
8
---
9
9
```

this violates the w3c get element text [spec](https://www.w3.org/TR/webdriver/#get-element-text).  `element.text` should return the `innerText` of the element.

**Comment by jesg** [comment]
fwiw i implemented this jesg/ghostdriver@b8fdce10c0bc94c1c46190ca8967fcfb84ea5ccd .  there where a few regressions in the upstream selenium regression tests so i did not merge this into my master branch.  the selenium atom trims off special characters and newlines.
